contrib/pnpinfo/pnpinfo.c

```diff
@@ -25,6 +25,7 @@
  */
 
 #include <sys/time.h>
+#include <sys/tslog.h>
```

Comment:
This file is userland code, and not used in boot at all. In fact, only really old i386 systems would have any cards this might find and it's likely going to be removed from the system when we remove i386 kernel support. What's the point of profiling it?